dataplane/standalone/saiserver/attrmgr/attrmgr.go

```diff
@@ -63,6 +68,27 @@ func (mgr *AttrMgr) set(id string, attr int32, val protoreflect.Value) {
 
 const protoNS = "lemming.dataplane.sai"
 
+// InvokeAndSave calls the RPC method and saves the attributes in the request.
+// This is the same behavior as the Interceptor, except for invoking server methods directly (not using gRPC).
+func InvokeAndSave[T proto.Message, S proto.Message](ctx context.Context, mgr *AttrMgr, rpc func(context.Context, T) (S, error), req T) (S, error) {
```

Comment:
Why are you using generics here?  Since T and S are both proto.Message and proto.Message is an interface can't you just use them directly?

Reply:
By using generics you can do
`InvokeAndSave(ctx,mgr, port.CreatePort, portReq)`
otherwise you'd have to do
```
InvokeAndSave(ctx, mgr, func(ctx context.Context, req proto.Message) (proto, error) {
   return port.CreatePort(ctx,req)
}, portReq)
```